hypothesis-python/src/hypothesis/strategies/_internal/strings.py

```diff
@@ -152,6 +153,8 @@ def filter(self, condition):
                 f"You applied str.{condition.__name__} as a filter, but this allows "
                 f"all nonempty strings!  Did you mean str.is{condition.__name__}?",
                 HypothesisWarning,
+                # stacklevel=17
```

Comment:
I just left these comments on files to show what value I found was necessary to point to my code in the test script mentioned in the PR description.

I'll remove them once a decision's made in https://github.com/HypothesisWorks/hypothesis/pull/3406/files#r922869040.